third_party/ti_simplelink_sdk/run_sdk_drivers_gen.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Texas Instruments Incorporated
+
+"""A wrapper to run the SDK makefiles for Platform driver generation
+Args:
+1. [TI Simplelink SDK Root]
+2. [Matter repository Root]
+"""
+
+import os
+import subprocess
+import sys
+
+SDK_ROOT = sys.argv[1]
+CHIP_ENV_ROOT = sys.argv[2]
```

Comment:
should this not add `.environment` by default? Or alternatively error out if PW_ACTUAL_ENVIRONMENT_ROOT does not exist.

Reply:
Thanks for the feedback, updated PR.